lib/win32/taskscheduler.rb

```diff
@@ -491,17 +491,24 @@ def priority=(priority)
     # trigger variable is a hash of options that define when the scheduled
     # job should run.
     #
-    def new_work_item(task, trigger)
+    def new_work_item(task, userinfo, trigger)
```

Comment:
where is this called from? just line 172? https://github.com/chef/win32-taskscheduler/pull/31/files#diff-6e8eaa39e1a37c92c78a5801c8d9a3abL172

Don't we need to update everywhere this is called from as well to pass the same arguments?

Reply:
@btm thank you. I have fixed the code to give backward compatibility. There is call for new_work_item from https://github.com/chef/win32-taskscheduler/blob/ole/lib/win32/taskscheduler.rb#L172 which now does not require to do any changes since user information is now optional argument. I have related changes in the window_task PR here https://github.com/chef/chef/pull/7288 which needs to be merged after this gets merged.